hack/lib/util/golang.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# This library holds golang related utility functions.
+
+# Ensure the go tool exists and is a viable version.
+os::golang::verify_go_version() {
+  if [[ -z "$(which go)" ]]; then
+    cat <<EOF
+Can't find 'go' in PATH, please fix and retry.
+See http://golang.org/doc/install for installation instructions.
+EOF
+    return 1
+  fi
+
+  local go_version
+  go_version=($(go version))
+  if [[ "${go_version[2]}" < "go1.6" && "${go_version[2]}" != "devel" ]]; then
```

Comment:
Ok, I'll rework it in such a way that it will always fail for != go1.6, not like currently if go1.6 is not there we'll just skip this over and you never see a failure. But I'll provide a hint in the error message how to bypass the problem.